lib/tasks/generate-from-blueprint.js

```diff
@@ -12,8 +12,52 @@ module.exports = Task.extend({
   blueprintFunction: 'install',
 
   run: function(options) {
-    var self = this;
-    var name = options.args[0];
+    var blueprints = [];
+    var firstArg = options.args[0];
+
+    var paths = Blueprint.defaultLookupPaths();
+    paths = paths.concat(this.project.localBlueprintLookupPath());
+
+    var isInPaths = Blueprint.isFoundInPaths(firstArg, paths);
+
+    if (isInPaths) {
+      blueprints.push({
+        name: firstArg,
+        args: options.args,
+        options: options
+      });
+    } else {
+      options.args.forEach(function(arg) {
+        blueprints.push({
+          name: arg,
+          options: options
+        });
+      }, this);
+    }
+
+    return this.runBlueprints(blueprints);
+  },
+
+  lookupBlueprint: function(name, ignoreMissing) {
+    return Blueprint.lookup(name, {
+      paths: this.project.blueprintLookupPaths(),
+      ignoreMissing: ignoreMissing
+    });
+  },
+
+  runBlueprints: function(blueprints) {
+    return this.runBlueprint(blueprints.shift())
+      .then(function() {
+        if (blueprints.length > 0) {
+          this.runBlueprints(blueprints);
```

Comment:
Ah yes I see. I will update to switch to using the sequence helper which will kill off this code anyway.